drivers/axi_core/spi_engine/spi_engine.h

```diff
@@ -86,6 +86,8 @@ and will be used inside the function call */
  * @brief  Structure containing the init parameters needed by the SPI engine
  */
 struct spi_engine_init_param {
+	/** SPI engine reference clock */
+	uint32_t	ref_clk_hz;
```

Comment:
Is there a way to detect this from the platform instead/in addition to the user providing it?

Reply:
If we have the guarantee that this parameter is named "XPAR_PS7_SPI_0_SPI_CLK_FREQ_HZ" always, yes we can use this. Should we use it?

Reply:
We can't guarantee this because it would depend on HDL notation. If there is no other way we should leave it like this, I think.